package.json

```diff
@@ -75,6 +74,7 @@
     "pull-pause": "0.0.2",
     "pull-pushable": "^2.2.0",
     "pull-stream": "^3.6.2",
+    "pull-through": "^1.0.18",
```

Comment:
This PR doesn't actually use `pull-through` - it appears to be a missing dependency as it's used by the [`builder`](https://github.com/ipfs/js-ipfs-unixfs-engine/blob/master/src/builder/builder.js#L6) and [`trickle-reducer`](https://github.com/ipfs/js-ipfs-unixfs-engine/blob/master/src/builder/trickle/trickle-reducer.js#L7) modules.